paper-server/src/main/java/org/bukkit/craftbukkit/inventory/view/builder/CraftDoubleChestInventoryViewBuilder.java

```diff
@@ -15,6 +16,7 @@ public class CraftDoubleChestInventoryViewBuilder<V extends InventoryView> exten
 
     public CraftDoubleChestInventoryViewBuilder(final MenuType<?> handle) {
         super(handle);
+        super.defaultTitle = Component.translatable("container.chestDouble");
```

Comment:
Both these should get a `// Paper - diff on change - CraftDoubleChestInventoryViewBuilder.defaultTitle` on the respective bits in NMS that "define" them / use them